### PR TITLE
added a 'no results' message

### PR DESCRIPTION
**Pull request by devonbNU**

### CODE REVIEW

File: src/css/finder-stylesheet.css [hunk 1]
```diff
@@ -3,6 +3,9 @@
 	float: right;
 }
 
+.ul-container-no-filter {
```
Comment (jholm117) [inline src/css/finder-stylesheet.css]: what is this change for?

File: src/css/finder-stylesheet.css [hunk 2]
```diff
@@ -14,6 +17,12 @@ body{
     background-color: rgb(212, 232, 233); 
 }
 
+.no-results{
+	font-size: 24pt;
+	z-index: -1;
+	position: absolute;
```
Comment (jholm117) [inline src/css/finder-stylesheet.css]: position absolute and z index looks a bit janky, why not just center the text in a div?

File: src/css/finder-stylesheet.css (hunk 1 above)
Comment (devonbNU) [reply src/css/finder-stylesheet.css]: On the recipes page, the ul is in a container that allowed me to place the text so it sits behind whatever first element in the list is. Needed to make an analogous container for the shopping list. Doesn't really need css though, I can delete it

File: src/css/finder-stylesheet.css (hunk 2 above)
Comment (devonbNU) [reply src/css/finder-stylesheet.css]: z index and position absolute are needed to make the text sit behind the first item in the list

**Pull request merged**